src/components/__snapshots__/Text.test.js.snap

```diff
@@ -10,26 +10,26 @@ exports[`does not render markdown images 1`] = `
       },
       "h1": Object {
         "fontFamily": "Poppins-Bold",
-        "fontSize": 24,
+        "fontSize": 12,
```

Comment:
I'm a little concerned that by default these get returned as half of the intended font size in our tests. Is there a reason this is happening?